.github/workflows/continuous-build.yml

```diff
@@ -20,42 +20,52 @@ jobs:
 
   lint-test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18, 20, latest]
```

Comment:
Figured we might as well test on multiple Node versions since there a several different versions across API3 projects